src/main/java/org/folio/rest/impl/FeeFineActionsAPI.java

```diff
@@ -164,20 +163,14 @@ private Response sendPatronNoticeIfNeedBe(Feefineaction entity, Map<String, Stri
 
   private Future<Response> publishLogEvent(Feefineaction entity, Map<String, String> okapiHeaders,
     Context vertxContext, Response response) {
-
-    // do not publish log records for CREDIT and REFUND actions
-    if (!CREDIT.isActionForResult(entity.getTypeAction()) &&
```

Comment:
Why was this if-block initially presented? Does this conflict with the original business requirements statement? CC @siarhei-charniak